recipes/xcb-util/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "xcb-util" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://xcb.freedesktop.org/dist/xcb-util-{{ version }}.tar.bz2
+  sha256: 46e49469cb3b594af1d33176cd7565def2be3fa8be4371d62271fabb5eae50e9
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('xcb-util', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - libxcb >=1.13
+    - xorg-xproto
+  run:
+    - libxcb >=1.13
```

Comment:
This should be something like,
```
- {{ pin_compatible("libxcb", max_pin=None) }}
```
for the correct lower bound.

Reply:
Ahhh sure. This makes for less maintenance and only one version in the recipe. Ideally libxcb would have a run export.

Reply:
This is not for maintenance. This is for correctness.
For example, xcb-util might be built against 1.14, but the run is still >=1.13, but we want it to be >=1.14

Reply:
Ah I see your point. This assumes something might happen with the solver picking two different versions, which is totally possible. Easy to clean up in the feedstock.